src/main/scala/scorex/utils/Time.scala

```diff
@@ -69,7 +69,10 @@ class TimeImpl extends Time with ScorexLogging with AutoCloseable {
       case Some((server, newOffset)) if !scheduler.isShutdown =>
         log.trace(s"Adjusting time with $newOffset nanoseconds, source: ${server.getHostAddress}.")
         offset = newOffset
-        updateTask.delayExecution(ExpirationTimeout)
+        val cntSysTime = System.currentTimeMillis()
```

Comment:
Why not use ```correctedTime()``` here?
Minting task starts at a correctedTime in my opinion. Such as the offset is just about -500 ms

Reply:
1. right, it should be `correctedTime()` here.

2. we may further pick out the `500ms` parameter out to `conf` file and let user to decide the sync time point in this period. so, in this case, slot 0 minter can set it to 30000ms.